readthedocs/docsitalia/management/commands/import_allowed_tags.py

```diff
@@ -0,0 +1,15 @@
+"""
+Command to sync allowed tags
+"""
+from django.core.management.base import BaseCommand
+
+from readthedocs.docsitalia.models import AllowedTag
+from readthedocs.docsitalia.tags_vocabulary import BASE_TAGS
+
+
+class Command(BaseCommand):
+    help = "Sync allowed tags on the database with the python one."
+
+    def handle(self, *args, **options):
+        for tag in BASE_TAGS:
+            AllowedTag.objects.get_or_create(name=tag.lower(), defaults={'enabled': True})
```

Comment:
Si potrebbe specificare un classmethod nel modello AllowedTag da invocare poi qui, in modo che sia riutilizzabile anche nelle migrazioni o in altri punti del codice.

Reply:
gli historical model non hanno metodi custom (https://docs.djangoproject.com/en/2.2/topics/migrations/#historical-models) quindi non si può usare un classmethod. si potrebbe creare una funzione a cui passare il model, ma dal momento che usiamo solo `get_or_create` non vedo problemi

Reply:
Hai ragione! Avendo riletto bene la documentazione a maggior ragione eviterei assolutamente di far dipendere la migrazione da codice esterno che potrebbe cambiare. In questo caso poi come hai detto tu usiamo solo un `get_or_create` quindi la duplicazione mi sembra trascurabile a fronte di codice più consistente.

Reply:
va bene